### PR TITLE
Increment TileDB-Vector-Search version to 0.0.20

### PR DESCRIPTION
**Pull request by dudoslav**
Increment version of TileDB-Vector-Search to 0.0.20.

This PR still needs to wait until TileDB-Vector-Search is actually updated to this new version.

### CODE REVIEW

**Comment by dudoslav** [comment]
Also re-render might be in place

**Comment by jdblischak** [comment]
I found the problem! The commit to build TileDB-Py 0.25.0 with tiledb 2.19 failed only for the win-64 py39 job!

image

https://github.com/conda-forge/tiledb-py-feedstock/commit/4ff96c1c149e6cea6b456dd10f6b188941e3437e

**Comment by jdblischak** [comment]
And for a post-mortem, the strange solver error about the outdated lz4-c is because the solver was choosing a super old version of tiledb-py to install. I don't know why it is choosing this one, but the actual problem was that none was suitable.

I discovered this by installing an env on Windows using the specs I posted above. Then in that env I tried to install `tiledb=2.19`. The lz4-c 1.8.1 was in an attempt to install tiledb-py 0.3.1. Also note that the results are slightly confusing since it says that tiledb-py 0.25.0 needs to be installed with tiledb 2.18, but that is just an artifact of https://github.com/conda-forge/tiledb-py-feedstock/pull/191#issuecomment-1885658469, which we are in the process of cleaning up.


```sh
mamba env create --file vector-search.yaml
conda activate vector-search
mamba install --dry-run tiledb=2.19
## │  ├─ tiledb-py 0.3.1 would require
## │  │  └─ tiledb 1.4.* , which requires
## │  │     └─ lz4-c >=1.8.1.2,<1.8.2.0a0 , which requires
## │  │        └─ vc 14 , which does not exist (perhaps a missing channel);
## └─ tiledb 2.19**  is not installable because it conflicts with any installable versions previously reported.
mamba install --dry-run tiledb=2.19 tiledb-py=0.25
## Could not solve for environment specs
## The following packages are incompatible
## ├─ tiledb-py 0.25**  is installable with the potential options
## │  ├─ tiledb-py 0.25.0 would require
## │  │  └─ tiledb >=2.18.3,<2.19.0a0 , which can be installed;
## │  ├─ tiledb-py 0.25.0 would require
## │  │  └─ python >=3.10,<3.11.0a0 , which can be installed;
## │  ├─ tiledb-py 0.25.0 would require
## │  │  └─ python >=3.11,<3.12.0a0 , which can be installed;
## │  └─ tiledb-py 0.25.0 would require
## │     └─ python >=3.8,<3.9.0a0 , which can be installed;
## └─ tiledb 2.19**  is not installable because it conflicts with any installable versions previously reported.
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,19 +35,21 @@ requirements:
     - python
     - numpy
     - pybind11
-    - tiledb >=2.18.0,<2.19
+    - tiledb >=2.19,<2.20
     - tiledb-py
     - scikit-build
     - libopenblas
     - openblas
+    - lz4-c 1.9.4 # [win and py==39]
```
Comment (jdblischak) [inline recipe/meta.yaml:43]: You can remove this now

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,19 +35,21 @@ requirements:
...
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - scikit-learn
-    - tiledb >=2.18.0,<2.19
+    - tiledb >=2.19,<2.20
     - tiledb-py
     - libopenblas
     - openblas
+    - lz4-c 1.9.4 # [win and py==39]
```
Comment (jdblischak) [inline recipe/meta.yaml:52]: and this

**Review by jdblischak — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (dudoslav) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 1 above)
Comment (dudoslav) [reply recipe/meta.yaml]: done

**Review by jdblischak — APPROVED** [review]

**Pull request merged**